dGame/dComponents/ControllablePhysicsComponent.cpp

```diff
@@ -15,7 +15,7 @@
 #include "LevelProgressionComponent.h"
 #include "eStateChangeType.h"
 
-ControllablePhysicsComponent::ControllablePhysicsComponent(Entity* entity) : PhysicsComponent(entity) {
+ControllablePhysicsComponent::ControllablePhysicsComponent(Entity* entity, int32_t componentId) : PhysicsComponent(entity, componentId) {
```

Comment:
If this is a component ID, shouldn't it use the corresponding enum type?
```suggestion
ControllablePhysicsComponent::ControllablePhysicsComponent(Entity* entity, eReplicaComponentType componentId) : PhysicsComponent(entity, componentId) {
```

Reply:
It's the id of that component entry, not the type of component

Reply:
What's the difference? Sorry if this is a dumb question.

Reply:
component type is like the class type, component id is the row in the db to query